rpcserver.go

```diff
@@ -6745,6 +6751,222 @@ func (r *rpcServer) ListPermissions(_ context.Context,
 	}, nil
 }
 
+func (r *rpcServer) updateFeatures(updates []*lnrpc.UpdateFeatureAction) (*lnrpc.Op, error) {
+
+	ops := &lnrpc.Op{Entity: "features"}
+
+	fsets := make(map[feature.Set]*lnwire.RawFeatureVector)
+	sets := []feature.Set{feature.SetNodeAnn}
```

Comment:
@Roasbeef should the endpoint allow the user to update ANY feature set? (I think yes... but I am not sure)
In that case, we should include it in the proto request (set-bit-action)

Reply:
Yeah I think _any_ feature, other than features we have set by default (existing in the feature deps set).